### ISSUE TITLE
Unable to do `df` motion with ideavim

### ISSUE THREAD

**Issue by megawubs** [acejump/AceJump#397]
**Describe the bug**
Unable to do `df` motion with ideavim.

**To Reproduce**
1. Install & Enable the AceJump plugin
2. Add  the following to .ideavimrc:
```
" Press `f` to activate AceJump
map f :action AceAction<cr>
" Press `F` to activate Target Mode
map F :action AceTargetAction<cr>
" Press `g` to activate Line Mode
map g :action AceLineAction<cr>
```
(taken from the readme)
4. Go to a file with some text. Hit `df` and see `action AceAction` being pasted on the current cursor position

**Expected behavior**
I expect to be able to use the `df[char]` vim motion to delete forward to a given char.

**Screenshots**
![image](https://user-images.githubusercontent.com/2302867/154653040-eadec98a-c46c-492c-99a9-ed70174d6448.png)

**Desktop (please complete the following information):**
 - OS: macOS Monterey
 - IDE and version: PHPStorm 2021.3.2
 - AceJump version: 3.8.5

**Comment by breandan** [comment]
Thank you for reporting. The documentation for Vim bindings should be improved, but I don't think this is a bug with AceJump specifically. If you want to delete to a character using the provided `.ideavimrc` configuration, first type <kbd>f</kbd> to activate AceJump, then search for some text. When it comes time to make the final jump, use <kbd>Shift</kbd>, then complete the tag to select all text from the current cursor position to that destination. Finally, press <kbd>x</kbd> to delete the text. Does that make sense?

**Comment by breandan** [comment]
Yes, that is odd. I can reproduce, but I think it might be an issue for [IdeaVim](https://youtrack.jetbrains.com/issues/VIM). Perhaps we could suggest better defaults. There are [other mappings](https://stackoverflow.com/questions/3776117/what-is-the-difference-between-the-remap-noremap-nnoremap-and-vnoremap-mapping) that only apply in certain modes.

**Comment by chylex** [comment]
I did implement support (or hack) for supporting IdeaVim with `d/c/y` followed by AceJump commands in my fork: https://github.com/chylex/IntelliJ-AceJump/blob/vim/src/main/kotlin/org/acejump/vim/AceVimHandler.kt

If there's interest, I can look into adding support into upstream AceJump, though having support from IdeaVim's side sounds like a better idea, so maybe I could look into a PR for IdeaVim first.
